src/pages/documentation/resources/upgrade-platform-sdks/v4-faq.md

```diff
@@ -0,0 +1,29 @@
+# Frequently asked questions - Adobe Mobile Library (v4) end-of-support
+
+## Why is the support for Adobe Mobile Library (v4) ending?
+
+The version 4 SDKs were first released in September 2013 to support mobile app implementations for Adobe Analytics (including Mobile Services), Adobe Audience Manager, Adobe Target, and Adobe Campaign. These performant, well-tested SDKs provided rich functionality and are used in well over 18,000 apps across app stores.
+
+Spurred by customer feedback for more control, performance, and functionality over the years; and to support future product enhancements, we released the re-architected [Adobe Experience Platform Mobile SDKs](../../../index.md) in September 2018. The new Experience Platform Mobile SDKs replaces the version 4 SDKs and provide support for new features such as flexible configuration through the Data Collection UI, modular Adobe and partner extensions.
+
+Solutions such as [Places Service](https://experienceleague.adobe.com/docs/places/using/home.html) and [Adobe Experience Platform Assurance](https://experienceleague.adobe.com/docs/experience-platform/assurance/home.html) are exclusively available on Experience Platform Mobile SDKs.
+
+## Where do I learn more about upgrading/migrating to the Experience Platform Mobile SDKs?
+
+To upgrade to the Experience Platform SDKs, see [Upgrade to the Experience Platform SDKs](..).
```

Comment:
Is this link correct?

Reply:
this was the only way I was able to point to the local index.md, seems to work on localhost